kubernetes/callback.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
```

Comment:
Any ideas how to handle systems where Python2 and Python3 are installed in parallel and we want to install Patroni and dependencies on Python3. Currently I just had the Dockerfile sed in the correct version, but that feels a bit hacky...

Reply:
Not really.
We just call such scripts not as executables, but with specifying interpreter in front: `python3 /callback.py`

Reply:
Of course. Just specifying it in the config makes much more sense than patching the file itself.